container-toolkit/cdi-support.md

```diff
@@ -39,8 +39,12 @@ CDI also improves the compatibility of the NVIDIA container stack with certain f
 
 ### Procedure
 
-Two typical locations for CDI specifications are `/etc/cdi/` and `/var/run/cdi`.
-However, the path to create and use can depend on the container engine that you use.
+Two common locations for CDI specifications are `/etc/cdi/` and `/var/run/cdi/`.
+NVIDIA recommends using the `/etc/cdi/` directory for user-applied changes, such as performing the following steps.
+The `/var/run/cdi/` directory is commonly used for dynamic changes made by software.
+The contents of the `/var/run/cdi/` directory can change at runtime and can be deleted after a reboot.
```

Comment:
Using "can be deleted" here seems to imply an action by the user. It is also not clear what "can change at runtime" implies.